testsuite/domain/src/test/resources/host-configs/host-slave-rbac.xml

```diff
@@ -78,7 +78,7 @@
 
     <domain-controller>
         <!-- Remote domain controller configuration with a host and port -->
-        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm">
+        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false">
```

Comment:
Similar comment as for host-slave-rbac-properties.xml.

The 3 testsuites that use this config pass without this change.